man/check_mask.txt

```diff
@@ -0,0 +1,178 @@
+
+*--ie* _ie(1),...,ie(nobc)_::
+Ending i-index for open boundary condition. The index is Fortran index.
+
+*--js* _js(1),...,js(nobc)_::
+Starting j-index for open boundary condition. The index is Fortran index.
+
+*--je* _je(1),...,je(nobc)_::
+Ending i-index for open boundary condition. The index is Fortran index.
+
+*--model* _model_::
+Specify the component model. Its value could be _ocean_ or _land_. The default
+value is _ocean_.
+
+EXAMPLES
+--------
+
+. Use mask domain in fully coupled model for ocean
++
+----
+$ check_mask --grid_file ocean_mosaic.nc \
+   --ocean_topog ocean_topog.nc \
+   --min_pe 100 --max_pe 200 --model ocean
+----
++
+This will create a list of mask_table with filename
+_mask_table.n_mask.layout(1)Xlayout(2)_, where _n_mask_ is the number of masked
+domain processors, _layout(1)_ is the x layout value and _layout(2)_ is the y
+layout value.  For example the file "mask_table.9.18x10" has layout=(18,10) and
+9 domain processor is masked out.
++
+To use this file in a FMS run, place the mask_table file in the INPUT directory.
+Set ocean_model_nml layout variable to "layout(1),layout(2)", and the mask_table
+variable to "INPUT/mask_table_file".  Set the coupler_nml ocean_npes variable to
+(layout(1) x layout(2) - n_mask).  For the "mask_table.9.18x10" file example,
+ocean_model_nml/layout = 18,10; ocean_model_nml/mask_table =
+"INPUT/mask_table.9.18x10"; and coupler_nml/ocean_npes = 171.  (171 = 18*10-9).
++
+NOTE: For coupled models using concurrent coupling, you may only wish to set
+ocean_model_nml variable mask_table, and not set ice_model_nml mask_table. The
+reason for avoiding the ice_model use of mask_table with fully coupled models is
+that, 1. the mask_table for the sea ice may not be necessary for performance
+enhancement in fully coupled models, and 2. most importantly, setting mask_table
+for sea ice complicates the layout for the ice model in fully coupled models.
+
+. Use mask domain in sea-ice model (baltic1 experiment, ocean, ice,
+atmosphere and land are all on the same grid)
++
+----
+$ check_mask --grid_file baltic1_grid_spec.nc \
+   --min_pe 60 --max_pe 80
+----
++
+This will create a list of mask_table with filename
+mask_table.n_mask.layout(1)Xlayout(2). For example mask_table.26.6x11 has
+layout=(6,11) and 26 domain processor is masked out. As in example 1, place the
+file in the INPUT directory, set ocean_model_nml/layout = 6,11, and
+ocean_model_nml/mask_table = "INPUT/mask_table.26.6x11".  Since this model has
+the ocean, ice, atmosphere and land on the same grid, set these same variables
+to the same value in ice_model_nml, atmos_model_nml, and land_model_nml.  The experiment
+will run on 40 processors (6*11-26).
+
+BUGS
+----
+Open bug reports at {package_bugreport}.
+
+AUTHORS
+-------
+Zhi Liang
+
+COPYRIGHT AND LICENSE
+---------------------
+Copyright {copyright_year} Geophysical Fluid Dynamics Laboratory
+
+The Flexible Modeling System (FMS) is free software; you can
```

Comment:
I updated many of the locations that have this.  Places I didn't update this are items that came directly from FMS, and should keep the same package name.  It may be confusing, but that is why they weren't changed.